frontend/ui/data/css_tweaks.lua

```diff
@@ -0,0 +1,166 @@
+                separator = true,
+            },
+            {
+                id = "a_bold";
+                title = _("Links always bold"),
+                css = [[a { font-weight: bold !important; }]],
+            },
+            {
+                id = "a_not_bold";
+                title = _("Links never bold"),
+                css = [[a { font-weight: normal !important; }]],
+            },
+        },
+    },
+    {
+        title = _("Misc"),
+        {
+            id = "table_row_odd_even";
+            title = _("Alternate background color of table rows"),
+            css = [[
+tr:nth-child(odd)  { background-color: #EEE !important; }
+tr:nth-child(even) { background-color: #CCC !important; }
+            ]],
+        },
+        {
+            id = "table_force border";
+            title = _("Show borders on all tables"),
+            css = [[
+table, tcaption, tr, th, td { border: black solid 1px; border-collapse: collapse; }
+            ]],
+            separator = true,
+        },
+        {
+            id = "image_full_width";
+            title = _("Make images full width"),
+            description = _("Useful for books containing only images, when they are smaller than your screen. May stretch images in some cases."),
+            -- This helped me once with a book. Will mess with aspect ratio
+            -- when images have a style="width: NNpx; heigh: NNpx"
+            css = [[
+img {
+   text-align: center !important;
+   text-indent: 0px !important;
+   display: block !important;
+   width: 100% !important;
+}
+            ]],
+        },
+    },
+    {
+        title = _("Workarounds"),
+        {
+            id = "html_tags_fix";
+            title = _("Fix some HTML elements"),
+            description = _("Make some HTML elements (eg: <cite>) behave as they should (inline/block).\nThis may break past bookmarks and highlights."),
+            css = [[
+cite { display: inline; font-style: italic; }
+            ]],
+        },
+        {
+            id = "list_items_fix";
+            title = _("Fix some list items issues"),
+            description = _("Work around some crengine list items rendering issues "),
+            css = [[
+li > p:first-child   { display: inline !important; }
+li > div:first-child { display: inline !important; }
+            ]],
+        },
+        {
+            id = "border_all_none";
+            title = _("Remove all borders"),
+            description = _("Work around a crengine bug that makes a border drawn when {border: black solid 0px}"),
+            -- css = [[* { border-style: none !important; }]],
+            -- Better to keep the layout implied by width, just draw them in white
+            css = [[* { border-color: white !important; }]],
+        },
+        {
+            id = "workaround_all_bugs";
+            title = _("Work around all bugs"),
+            description = _("Seriously ? :-)"),
+            css = [[
+body { display: none !important; }
```

Comment:
lol what :-P

Reply:
Yes, and even this workaround is buggy (it makes a 0 page book, and you reload the book with this tweak applied, you get a black page. So it's dangerous.
I'll remove it, unless we find some other funny and gentle style tweak.